Library/Formula/gitlab-ci-multi-runner.rb

```diff
@@ -16,21 +18,44 @@ class GitlabCiMultiRunner < Formula
   end
 
   depends_on "go" => :build
+  depends_on "godep" => :build
 
-  def install
-    ENV["GOPATH"] = buildpath
+  go_resource "github.com/jteeuwen/go-bindata" do
+    url "https://github.com/jteeuwen/go-bindata.git",
+      :revision => "a0ff2567cfb70903282db057e799fd826784d41d"
+  end
 
+  resource "prebuilt.tar.gz" do
+    url "https://gitlab-ci-multi-runner-downloads.s3.amazonaws.com/v1.0.4/docker/prebuilt.tar.gz",
+      :using => :nounzip
+    sha256 "43dedd023672990e27289e97bf74f493576956418148f6227917b8511e8aadfd"
+  end
```

Comment:
@DomT4 Let me explain it again. 

- **gitlab-ci-multi-runner** provides a feature which required docker runtime
- **gitlab-ci-multi-runner** is carrying built-in docker images inside it's binary, it's job of [go-bindata](https://github.com/jteeuwen/go-bindata.git)

So what we have done here is

- Declare `docker` as a recommended runtime dependency.
- Track built-in docker images with `resource`